pkgs/by-name/ni/nickel/package.nix

```diff
@@ -51,6 +51,12 @@ rustPlatform.buildRustPackage rec {
 
   passthru.updateScript = nix-update-script { };
 
+  checkFlags = [
+    # https://github.com/tweag/nickel/blob/1.10.0/git/tests/main.rs#L60
+    # `cmd.current_dir(repo.path()).output()` errors with `NotFound`
```

Comment:
I think that fails because we have no repo, right?

Reply:
that's right

Reply:
```suggestion
    # fails because src is not a git repo
    # `cmd.current_dir(repo.path()).output()` errors with `NotFound`
```